app/templates/components/modals/add-user-role-modal.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="header">
+  {{t 'Add Role'}}
+</div>
+<div class="content">
+  <form class="ui form" autocomplete="off" {{action (optional submit) on='submit' preventDefault=true}}>
```

Comment:
Why are you not allowing the user to submit via the default way here ? Only for the event delete modal I asked you to not do it because we need the users to press the button correctly to prevent any accidents. Every other place we should support normal form submit events.